modules/report/src/main/ModReportFilter.scala

```diff
@@ -1,14 +1,38 @@
 package lila.report
 
 import lila.user.User
+import org.joda.time.DateTime
+
+case class TempThreshold(score: Int, lastUsed: DateTime)
 
 final class ModReportFilter {
 
   // mutable storage, because I cba to put it in DB
   private var modIdFilter = Map.empty[User.ID, Option[Room]]
+  // mutable storage, because I also cba to put it in DB
+  // Also, this value shouldn't be in the DB. If we ever move the other value to the
+  // db we should move this to a cookie or something. I intend
+  // for this to be a temporary cookie of some sort.
+  private var modTempThreshold = Map.empty[User.ID, Option[TempThreshold]]
```

Comment:
this could be a `Cache[User.ID, Int]` where scaffeine handles the expiration.

there are many examples in lila, here's one https://github.com/ornicar/lila/blob/66213a0fd8ccab54a790ded4b92a14d8eb9c52b9/modules/security/src/main/Flood.scala#L16-L18

Reply:
Ah, perfect. Thanks.